.buildkite/pipeline.yaml

```diff
@@ -76,34 +76,24 @@ steps:
       - docker#v3.0.1:
           image: "matrixdotorg/riotweb-ci-e2etests-env:latest"
           propagate-environment: true
+          workdir: "/workdir/matrix-react-sdk"
 
   - label: "🔧 Riot Tests"
     agents:
       # We use a medium sized instance instead of the normal small ones because
       # webpack loves to gorge itself on resources.
       queue: "medium"
     command:
-      # Install chrome
-      - "echo '--- Installing Chrome'"
-      - "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
```

Comment:
Oh, I always though the karma tests relied still on this (e2e tests have puppeteer install it) :+1: